lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -1634,6 +1631,10 @@ std::string mangleBuiltin(StringRef UniqName, ArrayRef<Type *> ArgTypes,
                                               : (unsigned)BtnInfo->getVarArg();
          I != E; ++I) {
       auto T = ArgTypes[I];
+      auto MangleInfo = BtnInfo->getTypeMangleInfo(I);
+      if (MangleInfo.PointerTy && T->isPointerTy()) {
```

Comment:
Could there be situations where we encountered an opaque pointer type for which we don't have corresponding actual type within `MangleInfo`?

Reply:
Yes. `transTypeDesc` has an `if (Ty->isPointerTy())` block that treats such pointers as if they were `i8*`.